framework/src/modules/governance/stores/proposals.ts

```diff
@@ -25,4 +32,101 @@ export interface ProposalsStoreData {
 	description: ProposalDescription;
 }
 
-export class ProposalsStore extends BaseStore<ProposalsStoreData> {}
+export const proposalDescriptionSchema = {
+	type: 'object',
+	required: ['title', 'author', 'summary', 'discussionsTo', 'text'],
+	properties: {
+		title: {
+			dataType: 'bytes',
+			minLength: 1,
+			maxLength: MAX_LENGTH_PROPOSAL_TITLE,
+			fieldNumber: 1,
+		},
+		author: {
+			dataType: 'bytes',
+			minLength: 1,
+			maxLength: MAX_LENGTH_PROPOSAL_AUTHOR,
+			fieldNumber: 2,
+		},
+		summary: {
+			dataType: 'bytes',
+			minLength: 1,
+			maxLength: MAX_LENGTH_PROPOSAL_SUMMARY,
+			fieldNumber: 3,
+		},
+		discussionsTo: {
+			dataType: 'bytes',
+			maxLength: MAX_LENGTH_PROPOSAL_LINK,
+			fieldNumber: 4,
+		},
+		text: {
+			maxLength: MAX_LENGTH_PROPOSAL_TEXT,
+			fieldNumber: 5,
+			dataType: 'bytes',
+		},
+	},
+};
+
+export const proposalSchema = {
+	$id: 'governance/store/proposal',
```

Comment:
```suggestion
	$id: '/governance/store/proposal',
```

Not sure if it impacts functionality, but just to follow existing pattern.